api/src/main/java/edu/cornell/mannlib/vitro/webapp/application/VitroHomeDirectory.java

```diff
@@ -32,12 +38,16 @@ public static VitroHomeDirectory find(ServletContext ctx) {
 	private final ServletContext ctx;
 	private final Path path;
 	private final String discoveryMessage;
+    private Set<String> excludedHomeFiles = new HashSet<>(Arrays.asList("rdf"));
```

Comment:
Indentation should match file or style guide. I assume consistency would be preferred if varying from style guide.

Reply:
Fixed indentations.